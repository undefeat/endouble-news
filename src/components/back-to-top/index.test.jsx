import React from 'react';
import shallow from 'react-test-renderer/shallow';
import BackToTop from '.';

describe('BackToTop', () => {
    it('should correctly render', () => {
        const tree = shallow.createRenderer().render(<BackToTop />);

        expect(tree).toMatchSnapshot();
    });
});
