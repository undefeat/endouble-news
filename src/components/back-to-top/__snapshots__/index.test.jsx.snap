// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackToTop should correctly render 1`] = `
<button
  aria-label="Back to Top"
  className="back-to-top"
  hidden={true}
  onClick={[Function]}
  title="Back to Top"
  type="button"
>
  <i
    className="fas fa-arrow-up"
  />
</button>
`;
